### ISSUE TITLE
reaptcha does not support react 18

### ISSUE THREAD

**Issue by soasada** [sarneeh/reaptcha#248]
Hi, when I try to install this package in a fresh project with react 18.0.0 it throws the following error:

`npm i reaptcha`

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: frontend@0.1.0
npm ERR! Found: react@18.0.0
npm ERR! node_modules/react
npm ERR!   react@"18.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.6.3 || ^17.0.0" from reaptcha@1.11.0
npm ERR! node_modules/reaptcha
npm ERR!   reaptcha@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/nvortega/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/nvortega/.npm/_logs/2022-04-06T10_37_16_327Z-debug.log
```

Do you have plans to support it?

**Comment by DavideDA** [comment]
Same issue

**Comment by sarneeh** [comment]
Fixed in `1.12.0`
